# HG changeset patch
# User Ben Abbott <bpabbott@mac.com>
# Date 1230148225 18000
#      Wed Dec 24 14:50:25 2008 -0500
# Node ID b8ce8738a4d14df3d28539564c33ac3cd4959d93
# Parent  9038f311c1a8a07b39acaca13680615c8b2d28ce
chol.cc: Trivial fix for chol tests.

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,13 @@
+2008-12-24  Ben Abbott  <bpabbott@mac.com>
+
+	* DLD-FUNCTIONS/chol.cc: Compare results to 0 in tests.
+	Delete spurious function calls in test block.
+
 2008-12-24  John W. Eaton  <jwe@octave.org>
 
 	* DLD-FUNCTIONS/eigs.cc: Use "testif HAVE_ARPACK" to skip tests if
 	ARPACK is not available.
 
 	* Makefile.in (DLD_XSRC): Add eigs.cc to the list.
 
 	* load-save.cc (parse_save_options): Handle -double.
@@ -876,17 +881,17 @@ 2008-09-08  John W. Eaton  <jwe@octave.o
 
 	* ls-oct-ascii.cc (std::string extract_keyword (std::istream&,
 	const char *, const bool)): Also accept \r as line ending character.
 	* ls-oct-ascii.h (template <class T> bool extract_keyword
 	(std::istream&, const char *, T&, const bool)): Likewise.
 	(template <class T> bool extract_keyword (std::istream&, const
 	string_vector&, std::string&, T&, const bool)): Likewise.
 
-2008-09-03  Ben Abbott <bpabott@mac.com>
+2008-09-03  Ben Abbott  <bpabott@mac.com>
 
 	* mappers.cc: Use eps(single) as tolerance for single precesion test.
 
 2008-09-03  John W. Eaton  <jwe@octave.org>
 
 	* variables.cc (mlock): Lock current function on the call stack.
 	(Fmlock): Lock caller.
 
@@ -917,17 +922,17 @@ 2008-08-28  David Bateman  <dbateman@fre
 	* graphics.h.in (root_figure::properties::showhiddenhandles):
 	New property.
 
 2008-08-27  Jaroslav Hajek  <highegg@gmail.com>
 
 	* DLD-FUNCTIONS/__glpk__.cc (F__glpk__): Initialize output vectors 
 	by NA value.
 
-2008-08-26 Ben Abbott <bpabott@mac.com>
+2008-08-26 Ben Abbott  <bpabott@mac.com>
 
 	* mappers.cc: Adjust tolerance for single precesion tests.
 
 	* DLD-FUNCTIONS/chol.cc: Double tolerance.
 
 2008-08-26  Jaroslav Hajek  <highegg@gmail.com>
 
 	* parse.y (Fassignin): Use default  scope when calling varref.
diff --git a/src/DLD-FUNCTIONS/chol.cc b/src/DLD-FUNCTIONS/chol.cc
--- a/src/DLD-FUNCTIONS/chol.cc
+++ b/src/DLD-FUNCTIONS/chol.cc
@@ -468,27 +468,25 @@ symmetric positive definite matrix @var{
 
   return retval;
 }
 
 /*
 
 %!test
 %! A = [2,0.2;0.2,1];
-%! issymmetric(A)
-%! min(eig(A))
 %! Ainv = inv(A);
 %! Ainv1 = cholinv(A);
 %! Ainv2 = inv(sparse(A));
 %! Ainv3 = cholinv(sparse(A));
 %! Ainv4 = spcholinv(sparse(A));
-%! assert (norm(Ainv-Ainv1),1e-10)
-%! assert (norm(Ainv-Ainv2),1e-10)
-%! assert (norm(Ainv-Ainv3),1e-10)
-%! assert (norm(Ainv-Ainv4),1e-10)
+%! assert (norm(Ainv-Ainv1),0,1e-10)
+%! assert (norm(Ainv-Ainv2),0,1e-10)
+%! assert (norm(Ainv-Ainv3),0,1e-10)
+%! assert (norm(Ainv-Ainv4),0,1e-10)
 
 */
 
 DEFUN_DLD (chol2inv, args, ,
   "-*- texinfo -*-\n\
 @deftypefn {Loadable Function} {} chol2inv (@var{u})\n\
 Invert a symmetric, positive definite square matrix from its Cholesky\n\
 decomposition, @var{u}.  Note that @var{u} should be an upper-triangular\n\
